react/Autosuggest/Autosuggest.js

```diff
@@ -5,9 +5,16 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import ReactAutosuggest from 'react-autosuggest';
 import IsolatedScroll from 'react-isolated-scroll';
-import omit from 'lodash.omit';
+
+import get from 'lodash/get';
+import invoke from 'lodash/invoke';
+import omit from 'lodash/omit';
 
 import TextField from '../TextField/TextField';
+import smoothScroll from '../private/smoothScroll';
+
+const responsiveBreakpoint = '740';
+const smallDeviceOnlyMedia = `(max-width: ${responsiveBreakpoint - 1}px)`;
```

Comment:
This evaluates to `'740' - 1` — should probably make sure these are both numbers.